pycytominer/cyto_utils/cells.py

```diff
@@ -846,3 +775,67 @@ def aggregate_profiles(
             )
         else:
             return aggregated
+
+
+def _sqlite_strata_conditions(df, dtypes, n=1):
```

Comment:
should this be a separately defined function? When one imports `SingleCells` will it automatically know this function definition? I thought that you had it as a private method of the class before? Any reason you decided to make it separate?

Reply:
When one imports `SingleCells`, yes, a `SingleCells` object will automatically be able to see that function.  I did have it as a private method of the class before.  I can move it back if we want.  The only reason I moved it out is that I realized it doesn't actually rely on anything from `SingleCells` and it was just a tiny bit easier to import it for testing as a separate function.  Though it could still be imported for testing either way